### PR TITLE
Please review my work.

### PR DESCRIPTION
**Pull request by nithindeva**

### CODE REVIEW

File: nithin1/linux.txt [hunk 1]
```diff
@@ -1,3 +1,13 @@
+1. What is Linux?
+	a.Linux is a free os.
+	b.It is Unix like.
+	c.It is open source.
+	d.It is Network operating system
+2. Who developed the Linux?
+	Linux is developed by Linus Torvalds.
+3. How to get Linux?
+	Linux is available for downlode over the net, this is useful if your internet connection is fast. Otherwise we can use CD-ROM which saves time and is fast and automatic.
```
Comment (david1412) [inline nithin1/linux.txt]: See the indent,

File: nithin1/scrum.txt [hunk 2]
```diff
@@ -14,3 +14,4 @@ Scrum framework consists of scrum teams and their associated roles, events, arti
 USES OF SCRUM
 	Scrum has been used to develop software, hardware, embedded software, networks of interacting function, autonomous vehicles, schools, government, marketing, managing the operation of organizations and almost everything we use in our daily lives, as individuals and societies.
 	Scrum proved especially effective in iterative and incremental knowledge transfer.
+Scrum is a small team of people which is highly flexible and adaptive.
```
Comment (david1412) [inline nithin1/scrum.txt]: 5-10 points of scrum artefacts.

File: nithin1/git.txt [hunk 3]
```diff
@@ -28,4 +28,4 @@
 	Git is a distributed version control system.
 
 10. What are the interpreters for git in Mac machine?
-	terminal acts as the interpreter.
\ No newline at end of file
+	Terminal acts as the interpreter.
```
Comment (david1412) [inline nithin1/git.txt]: This file looks Good. Nice formating

**Review by david1412 — COMMENTED** [review]
Left some comments

**Review by pradeepm2406 — COMMENTED** [review]
Nice and crisp information.

**Review by pradeepm2406 — COMMENTED** [review]
About linux commands please add more points.

**Review by pradeepm2406 — COMMENTED** [review]
Can add the minimum and max team size number.

File: nithin1/scrum.txt (hunk 2 above)
Comment (nithindeva) [reply nithin1/scrum.txt]: I will add them in further process, please review the current work any doubts in this file.

File: nithin1/git.txt (hunk 3 above)
Comment (nithindeva) [reply nithin1/git.txt]: Thank you.

**Review by kulkarni90 — COMMENTED** [review]
looks short and  sweet.

Try to add more about scrum in next task as you said.

**Comment by prudhvidhargudimalla** [comment]
good one but don't repeat the same command lines in next task

File: nithin1/scrum.txt (hunk 2 above)
Comment (david1412) [reply nithin1/scrum.txt]: see the indent.Text formating

**Review by david1412 — COMMENTED** [review]
See text formating

File: nithin1/scrum.txt [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
+c).difficult to master.
 It is a process in which one can employ various processes and techniques.
 It helps in continuously improving the product, the team, and the working environment.
 Scrum framework consists of scrum teams and their associated roles, events, artifacts and rules.
 
 USES OF SCRUM
 	Scrum has been used to develop software, hardware, embedded software, networks of interacting function, autonomous vehicles, schools, government, marketing, managing the operation of organizations and almost everything we use in our daily lives, as individuals and societies.
 	Scrum proved especially effective in iterative and incremental knowledge transfer.
-Scrum is a small team of people which is highly flexible and adaptive.
+Scrum is a small team of people consisting of 3 to 9 people, which is highly flexible and adaptive.
```
Comment (david1412) [inline nithin1/scrum.txt:17]: Could you please write 5 artefacts and explain them.

**Review by david1412 — COMMENTED** [review]
Looks Good. Add the comments in scrum.txt then you are done. I'll approve it

File: nithin1/scrum.txt (hunk 4 above)
Comment (nithindeva) [reply nithin1/scrum.txt:17]: I thought of writing about artifacts in the next file

**Review by david1412 — APPROVED** [review]
looks Good.

**Review by pradeepm2406 — APPROVED** [review]
All looks good

**Pull request merged**